fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java

```diff
@@ -188,19 +189,18 @@ public boolean canCreate(String resourceType, Object resource) {
                       try {
                         fiatService.canCreate(username, resourceType, resource);
                         return true;
-                      } catch (RetrofitError re) {
+                      } catch (NotFoundException e) {
+                        return false;
+                      } catch (SpinnakerHttpException e) {
```

Comment:
We can omit setting to false but I think we still need to explicitly set retryable to true if the error code is not 400 or 404, right? [Because retryable is null by default](https://github.com/spinnaker/kork/blob/b6fa1103b7b3b12734af39cb6f6ca2b04cdd59d6/kork-exceptions/src/main/java/com/netflix/spinnaker/kork/exceptions/SpinnakerException.java#L45).

Reply:
oh, I just saw your other comment, so the behavior is the same whether retryable is set to null or true

Reply:
Right!